sbin/glite-info-static

```diff
@@ -29,18 +29,22 @@
 # WEB:          https://github.com/EGI-Federation/glite-info-static
 #
 ##############################################################################
-import os
-import sys
 import getopt
 import logging
+import os
+import sys
+
 
-# Funtion to print out the usage
 def usage():
-    sys.stderr.write('Usage: %s -m <module> [OPTIONS] \n' % (sys.argv[0]))
-    sys.stderr.write('''
-  glite-info-create.sh -m <module> [-i <ifaces>] [-t <templates>] [-c <configs>]
-                       [-p <path>] [-o <outpath>] [-d debug]
-       
+    """
```

Comment:
Lot of code could be just removed with argparse...

Reply:
Sure, it was a lazy/time-constrained update, if you feel like you would be able to do a PR for this it would great :)

Reply:
https://github.com/EGI-Federation/glite-info-static/pull/7